fluid-music/examples/groove-example.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const tab = require('../src/tab');
+const fluid = require('../src/index');
+const FluidClient = require('../src/FluidClient');
+const groove = require('../src/groove');
+
+const sessionPath = path.join(__dirname, 'sessions/out.tracktionedit')
+const drums909 = require('../../fluid-recipes/src/drumTrack909');
+const drumTrackName = '909';
+const drumsMsg = drums909(drumTrackName);
```

Comment:
One of the advantages of the new package structure is that these imports can be much cleaner. See how its done here:

https://github.com/CharlesHolbrow/cybr/blob/develop/fluid-recipes/examples/tab-parse.js#L3-L7